### PR TITLE
Added utility functions for the upcoming FTs

### PR DESCRIPTION
**Pull request by mish24**
### Checklist

- [x] I read the [contributing guide](/CONTRIBUTING.md) and I've followed it.
- [x] Commit message gives complete and useful information about the changes under it.
- [x] I ran the code locally and checked for common errors and uncaught exceptions.
- [ ] My code is unit testable.

After you submit your pull request, DO NOT close it or click 'Update Branch'.

Occasionally, you may be asked to rebase your branch.

This PR just contains the files and utility functions which will be used throughout our tests and are highly recurrent. These consist of page urls and their locators, the basic initiation needed for all tests , plus those functions which will be common for every test we run.

### CODE REVIEW

**Comment by mish24** [comment]
@prachi1210 That PR was very big, so I thought I'll first send one for the utility tests, and then next one for FTs.

File: MACCTests2017/tests/pom/pages/postsPage.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    def get_template_error_message(self):
+        return self.element_by_xpath(self.elements.TEMPLATE_ERROR_MESSAGE).text
+
+    def get_results(self):
+        return self.element_by_xpath(self.elements.RESULTS)
+
+    def get_pcsa_post_date(self):
+        return self.element_by_xpath(self.elements.PCSAPOST).text
+
+    def get_help_block(self):
```
Comment (prachi1210) [inline MACCTests2017/tests/pom/pages/postsPage.py]: @mish24 you already have a `get_help_blocks` function in your code.

Comment (mish24) [inline MACCTests2017/tests/pom/pages/postsPage.py]: Oh I just saw that. Actually the last one is supposed to be HELP_BLOCKS. I'll update it.

Comment (prachi1210) [inline MACCTests2017/tests/pom/pages/postsPage.py]: @mish24 Can you update this in this PR?

**Comment by prachi1210** [comment]
@mish24 instead of creating a separate commit, can you squash these commits into one? 
In future, please use `git commit --amend` 
If you want to do this for this PR
`git reset -soft HEAD~1` 
This does a soft reset for your changes while retaining them in the working directory. 
Now you can amend your previous commit.
Also, please go through the [contributing guide](/CONTRIBUTING.md) on how to write meaningful commit messages.

**Comment by mish24** [comment]
@prachi1210 I am not able to squash the commits. It won't show anything. Shall I send a new PR with the required changes?

**Comment by prachi1210** [comment]
@mish24 why don't you do a soft reset and then amend your commit? 
Sending a new PR is not the solution.

**Comment by prachi1210** [comment]
@mish24, all your PRs are being held up by this. Please update your PR

**Pull request closed without merging**